docs/drug_metabolite.rst

```diff
@@ -8,23 +8,58 @@ Drug metabolite
 Running
 ~~~~~~~
 
+The code to initiate structsearch for a drug metabolite model in Python/R is stated below:
+
+.. pharmpy-code::
+
+    from pharmpy.modeling import read_model
+    from pharmpy.tools read_modelfit_results, run_structsearch
+
+    start_model = read_model('path/to/model')
+    start_model_results = read_modelfit_results('path/to/model')
+    res = run_structsearch(type='drug_metabolite',
+                            search_space="METABOLITE([BASIC,PSC]);PERIPHERALS(0..1,MET)",
+                            model=start_model,
+                            results=start_model_results)
+
++-------------------------------------------------+---------------------------------------------------------------------+
+| Argument                                        | Description                                                         |
++=================================================+=====================================================================+
+| :ref:`type<the model types>`                    | Need to be set to 'drug_metabolite'                                 |
++-------------------------------------------------+---------------------------------------------------------------------+
+| :ref:`search_space<the search space>`           | Search space of models to test                                      |
++-------------------------------------------------+---------------------------------------------------------------------+
+| ``model``                                       | Start model                                                         |
++-------------------------------------------------+---------------------------------------------------------------------+
+| ``results``                                     | ModelfitResults of the start model                                  |
```

Comment:
Maybe ModelfitResults -> ``ModelfitResults`` so that it's clearer that it's an object type, or maybe write "The modelfit results of the start model"